dotnet/src/Experimental/Assistants/Experimental.Assistants.csproj

```diff
@@ -4,6 +4,7 @@
     <AssemblyName>Microsoft.SemanticKernel.Experimental.Assistants</AssemblyName>
     <RootNamespace>Microsoft.SemanticKernel.Experimental.Assistants</RootNamespace>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <VersionSuffix>alpha</VersionSuffix>
```

Comment:
This project is no longer in the solution and will be deleted soon. 

There is a new Experimental.Agents project to replace this, can you update that one instead.

Reply:
Good catch, thanks. Addressed!